### PR TITLE
Add scanner-name annotation to IVS

### PR DESCRIPTION
**Pull request by elfotografo007**
# Which other branches do you want a technical writer to cherry-pick this PR to (if any)?
This PR is targeting TAP 1.7

It's best to PR to the most recent relevant branch and leave all the cherry-picking to the
writer, except where earlier branches require factual changes to the content.
For more information about the branches, see https://github.com/pivotal/docs-tap#branches

### CODE REVIEW

**Comment by a-peek4** [comment]
Hi @elfotografo007 Let me know when you're ready to take this out of draft state. Thank you

File: tap-gui/troubleshooting.hbs.md [hunk 1]
```diff
@@ -426,6 +428,33 @@ overlays work with Tanzu Application Platform, see
    tanzu package installed update tap -p tap.tanzu.vmware.com -n tap-install --values-file tap-values.yaml
    ```
 
+### <a id='no-scanner-name'></a> Scanner name not shown
```
Comment (ryan-a-baker) [inline tap-gui/troubleshooting.hbs.md:431]:
```suggestion
### <a id='no-scanner-name'></a> Scanner name not shown in the Tanzu Developer Portal for Scan 2.0
```

File: scst-scan/ivs-trivy.hbs.md [hunk 2]
```diff
@@ -9,6 +9,8 @@ apiVersion: app-scanning.apps.tanzu.vmware.com/v1alpha1
 kind: ImageVulnerabilityScan
 metadata:
   name: trivy-ivs
+  annotations:
+    app-scanning.apps.tanzu.vmware.com/scanner-name: trivy
```
Comment (ryan-a-baker) [inline scst-scan/ivs-trivy.hbs.md:13]:
```suggestion
    app-scanning.apps.tanzu.vmware.com/scanner-name: Trivy
```

File: scst-scan/ivs-snyk.hbs.md [hunk 3]
```diff
@@ -18,6 +18,8 @@ apiVersion: app-scanning.apps.tanzu.vmware.com/v1alpha1
 kind: ImageVulnerabilityScan
 metadata:
   name: snyk-ivs
+  annotations:
+    app-scanning.apps.tanzu.vmware.com/scanner-name: snyk
```
Comment (ryan-a-baker) [inline scst-scan/ivs-snyk.hbs.md:22]:
```suggestion
    app-scanning.apps.tanzu.vmware.com/scanner-name: Snyk
```

File: scst-scan/ivs-prisma.hbs.md [hunk 4]
```diff
@@ -22,6 +22,8 @@ apiVersion: app-scanning.apps.tanzu.vmware.com/v1alpha1
 kind: ImageVulnerabilityScan
 metadata:
   name: prisma-ivs
+  annotations:
+    app-scanning.apps.tanzu.vmware.com/scanner-name: prisma
```
Comment (ryan-a-baker) [inline scst-scan/ivs-prisma.hbs.md:26]:
```suggestion
    app-scanning.apps.tanzu.vmware.com/scanner-name: Prisma
```

File: scst-scan/ivs-grype.hbs.md [hunk 5]
```diff
@@ -7,6 +7,8 @@ apiVersion: app-scanning.apps.tanzu.vmware.com/v1alpha1
 kind: ImageVulnerabilityScan
 metadata:
   name: grype-ivs
+  annotations:
+    app-scanning.apps.tanzu.vmware.com/scanner-name: grype
```
Comment (ryan-a-baker) [inline scst-scan/ivs-grype.hbs.md:11]:
```suggestion
    app-scanning.apps.tanzu.vmware.com/scanner-name: Grype
```

File: scst-scan/ivs-carbon-black.hbs.md [hunk 6]
```diff
@@ -20,6 +20,8 @@ apiVersion: app-scanning.apps.tanzu.vmware.com/v1alpha1
 kind: ImageVulnerabilityScan
 metadata:
   name: carbon-black-ivs
+  annotations:
+    app-scanning.apps.tanzu.vmware.com/scanner-name: carbon-black
```
Comment (ryan-a-baker) [inline scst-scan/ivs-carbon-black.hbs.md:24]:
```suggestion
    app-scanning.apps.tanzu.vmware.com/scanner-name: Carbon-Black
```

**Review by ryan-a-baker — APPROVED** [review]
Couple nit's on capitalization - but looks good to me.

**Pull request merged**